knockout-tinymce.coffee

```diff
@@ -7,7 +7,6 @@
     defaults: {}
     extensions: {}
     init: (element, valueAccessor, allBindings, viewModel, bindingContext) ->
-      throw "valueAccessor must be writeable and observable"  unless ko.isWriteableObservable(valueAccessor())
```

Comment:
We should replace this with an `Error` instance so that the stack trace is available on all browsers